### PR TITLE
Retail monitoring option

### PR DESCRIPTION
**Pull request by keichwa**
# Description

https://github.com/SUSE/spacewalk/issues/17497

# Target branches

Which documentation version does this PR apply to?
Does this content need to be backported? If this is the case, create an issue for tracking it and add the link to this PR.

# Links
- Related development PR #<insert PR link, if any>
- Fixes #<insert spacewalk issue, if any>

### CODE REVIEW

File: modules/retail/pages/retail-components.adoc [hunk 1]
```diff
@@ -20,13 +20,19 @@ They are used to securely build operating system images.
...
 == Branch Servers
 
 Branch servers should be physically located close to point-of-service terminals, such as in an individual store or branch office.
 Branch servers provide services for PXE boot, and act as an image cache, Salt broker, and proxy for software components (RPM packages).
 The branch server can also manage local networking, and provide DHCP and DNS services.
 
-
+[NOTE]
+====
+For monitoring, you can install Prometheus server on the Branch servers, and, for visualization and analysis, Grafana with multiple data sources on a dedicated host.
```
Comment (aaannz) [inline modules/retail/pages/retail-components.adoc:33]: I had to read the line multiple times to understand it. 😃 

Can it be split into two sentences, one for Prometheus and another for Grafana?

**Review by 0rnela — APPROVED** [review]

File: modules/retail/pages/retail-components.adoc (hunk 1 above)
Comment (0rnela) [inline modules/retail/pages/retail-components.adoc:33]:
```suggestion
* For monitoring, you can install Prometheus server on the Branch servers.
* For visualization and analysis, you can install Grafana with multiple data sources on a dedicated host.
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Comment by witekest** [comment]
The original issue was about Grafana not being available on the branch server. To that extend the change is good I think.

> Maybe @witekest can provide more details how to set it up and integrate with terminals.

The problem is we don't have a good procedure for discovering and configuring proxy clients. I have just created a [feature request](https://github.com/SUSE/spacewalk/issues/22756) for extending the current implementation.
Instead of documenting a workaround I'd prefer to wait until the new feature is implemented and then document it with all needed details.

File: modules/retail/pages/retail-components.adoc (hunk 1 above)
Comment (keichwa) [reply modules/retail/pages/retail-components.adoc]: Yes, I completely agree.  Thanks, Ornela, for the suggestion!

**Comment by keichwa** [comment]
Thanks for the background info, Witek!

**Pull request merged**